QUICKSTART.md

```diff
@@ -0,0 +1,165 @@
+# NgRX Auto-Entity
+
+Automatic entities for [@ngrx](https://github.com/ngrx/platform)! With this library, you can take the boiler off your plate, and
+get back to business!
+
+# Quick Start
+
+One of the primary goals with Auto-Entity is to provide a rapid development platform
+with minimal boilerplate requirements and easy setup. For those developers who wish
+to get rolling as quickly as possible, this guide should get you going fast!
+
+## Installation
+
+Install @briebug/ngrx-auto-entity from npm:
+
+`npm install @briebug/ngrx-auto-entity` or `yarn add @briebug/ngrx-auto-entity`
+
+## Setup
+
+### 1: App Module
+
+Add the `NgrxAutoEntityModule` to your app module:
+
+```typescript
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { NgrxAutoEntityModule } from '@briebug/ngrx-auto-entity';
+
+import { AppComponent } from './app.component';
+
+@NgModule({
+  declarations: [AppComponent],
+  imports: [BrowserModule, NgrxAutoEntityModule],
+  bootstrap: [AppComponent]
+})
+export class AppModule {}
+```
+
+### 2: Meta Reducer
+
+Add the autoEntityMetaReducer to your meta reducers for your application or
+feature:
```

Comment:
Do we want to suggest the file names for these snippets? e.g. `app.module.ts`, `app.reducer.ts`, etc?